### PR TITLE
Enable simple smoke test in the testing farm

### PR DESCRIPTION
**Pull request by psss**

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -21,3 +21,12 @@ jobs:
     - fedora-31-x86_64
     - fedora-rawhide-x86_64
   trigger: pull_request
+
+jobs:
```
Comment (TomasTomecek) [inline .packit.yaml]:
```suggestion
```
with this, you redefine jobs and thus remove copr-build :P no, packit is not taking a nap

Comment (psss) [reply .packit.yaml]: Oh! I see. Bad copy and paste. Thanks for spotting this.

**Comment by jpopelka** [comment]
just testing @packit-service/the-packit-team

**Pull request merged**